docs/en_US/guide/sinks/builtin/rest.md

```diff
@@ -2,21 +2,22 @@
+| insecureSkipVerify   | true     | Control if to skip the certification verification. If it is set to `true`, then skip certification verification; Otherwise, verify the certification. The default value is `true`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| oAuth                | true     | Define the authentication flow to follow the OAuth style. Other authentication method like apikey can directly set the key to header only, not need to set this configuration. Refer to [OAuth configuration](../../sources/builtin/http_pull.md#OAuth) in httppull source for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| compressionAlgorithm | true     | Use the specified compression algorithm to compress the payload. Upon receipt of a response, it automatically decompresses the server response using the specified algorithm. The current optional algorithms are: `zlib`, `gzip`, `flate`, `zstd`. **These are case sensitive**. Please note that this option will add the specified algorithm to the request header `Accept-Encoding`. You need to make sure that the server supports the algorithm you set. At the same time, you need to ensure that the server should use the same compression algorithm to compress the response data and set `Content-Encoding` in the response header. For the Flate algorithm, the `Accept-Encoding` request header value will be set to `deflate`. |
```

Comment:
To comply with other sink like mqtt, suggest to name it compression

Reply:
okay, working on it

Reply:
solved